recipes/hpcflow/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "hpcflow" %}
+{% set version = "0.2.0a50" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ba/93/57683655fc958b476d645d6f2d0dc5e2c431e7e827616e14b90ac8e49402/hpcflow_new2-0.2.0a50.tar.gz
```

Comment:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hpcflow-{{ version }}.tar.gz
```

Reply:
The reason why we used a  hardcoded link instead of the suggested code is that the coda-forge package name (hpcflow) is currently different to the PyPI package (hpcflow-new2). When the old version is deprecated in favour of the new version, we plan to use the suggested code, as the two names will match.